tomb-common/src/banyan_api/models/metadata.rs

```diff
@@ -231,6 +246,81 @@ pub mod test {
         let snapshot_id = metadata.snapshot(client).await?;
         Ok((metadata, host, authorization, snapshot_id))
     }
+
+    /// Simple struct for hosting the variables generated by this advanced test
+    #[derive(Debug)]
+    pub struct AdvancedTestSetup {
```

Comment:
We're probably gonna move stuff like this to an integration test in the future, and replace this type of stuff with stubs, but for now this stays here. This is a nice setup.